components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -246,6 +246,7 @@ protected void setUp() throws Exception {
         root = newRootOmeroClient();
         tmp.__del__();
 
+        scalingFactor = new Long(System.getProperty("omero.test.timeout"));
```

Comment:
Is it worth catching the case that omero.test.timeout property is not set (would throw a NPE at this point)?